addons/ai/fnc_taskPatrol.sqf

```diff
@@ -46,25 +51,30 @@ _position = _position call CBA_fnc_getPos;
 // Clear existing waypoints first
 [_group] call CBA_fnc_clearWaypoints;
 
+// Can pass paremeters straight through to addWaypoint
 private _this =+ _this;
```

Comment:
Something else.
`_this =+ _this`.
Or assign a new variable. Scope of `_this` already is set, so there is no point in that `private`.